distribution/docker/src/docker/config/oss/log4j2.properties

```diff
@@ -3,7 +3,7 @@ status = error
 appender.rolling.type = Console
 appender.rolling.name = rolling
 appender.rolling.layout.type = ECSJsonLayout
-appender.rolling.layout.type_name = server
+appender.rolling.layout.dataset = server
```

Comment:
Did not play around with the code itself. But if I think of the dataset, it is a unique name. So it should be `elasticsearch.server`. Does this make sense?

I'm assuming it follows the logic of the new naming scheme: https://www.elastic.co/blog/an-introduction-to-the-elastic-data-stream-naming-scheme but event.dataset should not be different.